samples/mixed/consumer/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+[package]
+name = "mixed-consumer"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+
+[dependencies]
```

Comment:
Consider my comment in the parent Cargo.toml.

Reply:
Fixed.